mindocr/data/transforms/det_transforms.py

```diff
@@ -99,6 +100,7 @@ def __call__(self, data):
 
     def _draw_border(self, np_poly, border, mask):
         # draw mask
+        np_poly = copy(np_poly)
```

Comment:
There's no need in explicit `copy`. This bug is fixed in [PR#204](https://github.com/mindspore-lab/mindocr/pull/204/files#diff-f2f94e4fbc850487c559769e5ca108c84f37d7f91c970e17a04e9b8cdc017836R110) but not merged yet.

Reply:
ok